### PR TITLE
review

### PR DESCRIPTION
**Pull request by nikob411**

### CODE REVIEW

File: exercises/pep8/name_generator.py [hunk 1]
```diff
@@ -1,15 +1,19 @@
+#!/usr/bin/env python
```
Comment (bharris47) [inline exercises/pep8/name_generator.py]: This is unnecessary unless you want to run this with a regular command invocation `./name_generator.py`. Just FYI.

**Comment by bharris47** [comment]
Also, work on providing better commit messages. `fixing PEP 8 issues` would be good enough.

File: exercises/pep8/name_generator.py (hunk 1 above)
Comment (nikob411) [reply exercises/pep8/name_generator.py]: good to know

**Comment by nikob411** [comment]
> Also, work on providing better commit messages. `fixing PEP 8 issues` would be good enough.
you got it

**Pull request merged**